runtime/browser/android/renderer_host/xwalk_render_view_host_ext.cc

```diff
@@ -105,16 +104,6 @@ void XWalkRenderViewHostExt::RenderProcessGone(base::TerminationStatus status) {
   }
 }
 
-void XWalkRenderViewHostExt::DidNavigateAnyFrame(
```

Comment:
@darktears Isn't Runtime only used in Linux/Windows? Per my understanding Android doesn't use Runtime.